src/redux/books/books.js

```diff
@@ -0,0 +1,22 @@
+// books actions
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:       
```
const ADD_BOOK = bookstore/books/ADD_BOOK 
const REMOVE_BOOK = bookstore/books/REMOVE_BOOK
```

Reply:
look at its result ``` boo-store/scr/redux/books/CHECK_STATUS ```